src/power_grid_model/_core/power_grid_model.py

```diff
@@ -120,6 +120,26 @@ def __deepcopy__(self, memo: dict[int, Any]) -> "PowerGridModel":
 
         return new_model
 
+    def __repr__(self) -> str:
+        """
+        Return a string representation of the current model.
```

Comment:
nitpick to follow our conventions regarding docstrings

```suggestion
        """Return a string representation of the current model.

```